lib/wallaby/session_store.ex

```diff
@@ -39,13 +39,13 @@ defmodule Wallaby.SessionStore do
   def handle_call({:demonitor, session}, _from, _state) do
     result =
       :ets.select(:session_store, [
-        {{{:"$1", :"$2", :"$3"}, :"$4"}, [{:==, :"$2", session.id}], [{{:"$1", :"$4"}}]}
-      ])
+        {{{:"$1", :"$2", :"$3"}, :"$4"}, [{:==, :"$2", session.id}], [{{:"$1", :"$3", :"$4"}}]}
+        ])
 
     case result do
-      [{ref, _}] ->
+      [{ref, pid, _session}] ->
```

Comment:
should this be `[{ref, _session, pid}]`?

Reply:
I should probably switch the return value order to match the way the data is stored in ETS. This call is based on the way the result is *returned from*, not *stored in* ETS (you can see the `:ets.delete` call matches the storage order.

Reply:
Oh, I see now that I misread it.